tests/spec/dirty_tests.py

```diff
@@ -173,6 +173,29 @@ def it_shows_error_on_stop_for_slow_start(self):
             norm=norm.pgctl,
         )
 
+    def it_restarts_fine_with_force_flag(self):
+        assert_command(
+            ('pgctl', 'start', 'sweet'),
+            '',
+            '''\
+[pgctl] Starting: sweet
+[pgctl] Started: sweet
+''',
+            0,
+        )
+        assert_command(
+            ('pgctl', 'restart', 'sweet', '--force'),
+            '',
+            '''\
+[pgctl] Stopping: sweet
+[pgctl] Stopped: sweet
+[pgctl] Starting: sweet
+[pgctl] Started: sweet
+''',
+            0,
+            norm=norm.pgctl,
+        )
```

Comment:
erm I think it's a success if you do what the user asked.
But showing a warning message is still appropriate.